src/lib/data/projects.ts

```diff
@@ -66,11 +67,14 @@ export async function getAllProjectsForPlan({
 
   const result = new Map<ProjectId, ProjectData>();
   for (const project of projects) {
-    if (!project.latestVersionId) {
+    if (!project.latestVersionId || !project.currentPublishedVersionId) {
       // Project only has ID and no base data yet, ignore
       continue;
     }
-    const projectVersion = pvsById.get(project.latestVersionId);
+    const projectVersion =
+      version == 'current'
```

Comment:
I am surprised that this wasn't caught by ESLint
```suggestion
      version === 'current'
```